metricbeat/mb/testing/fetcher.go

```diff
@@ -96,6 +96,8 @@ func (f *reportingMetricSetV2FetcherError) WriteEvents(t testing.TB, path string
 }
 
 func (f *reportingMetricSetV2FetcherError) WriteEventsCond(t testing.TB, path string, cond func(common.MapStr) bool) {
+	t.Helper()
```

Comment:
Just a heads up in case this line slipped here.

Reply:
This is intentionally added, so errors are more meaningful. This is not really related, so I can move this line to a different PR if you prefer.